### PR TITLE
fix(zeebe): delete calledDecision for <none> dmn implementation

### LINKED ISSUE #578: Empty Business Rule Task not Indicated as Empty
__Describe the Bug__

When creating a new `bpmn:BusinessRuleTask` it doesn't have a `zeebe:CalledDecision` or `zeebe:TaskDefinition` extension element by default. However, in the properties panel _DMN decision_ is preselected as the implementation type. This is misleading.

wkZv027AN0

__Steps to Reproduce__

1. Create new business rule task
2. Open the _Implementation_ group in the properties panel
3. _Implementation_ has _DMN decision_ selected

__Expected Behavior__

_&lt;none>_ is selected by default.

brave_BVi8PDHn4z

__Environment__

- Library version: 1.0.0-alpha.2

### PR DESCRIPTION
**Pull request by smbea**
Closes #578

### CODE REVIEW

File: src/provider/zeebe/properties/BusinessRuleImplementationProps.js [hunk 1]
```diff
@@ -76,6 +76,9 @@ function BusinessRuleImplementation(props) {
 
     let extensionElement, extensionElementType;
 
+    resetElement(element, commandStack);
+
+
```
Comment (MaxTru) [inline src/provider/zeebe/properties/BusinessRuleImplementationProps.js:81]:
```suggestion
```

File: src/provider/zeebe/properties/BusinessRuleImplementationProps.js [hunk 2]
```diff
@@ -76,6 +76,9 @@ function BusinessRuleImplementation(props) {
 
     let extensionElement, extensionElementType;
 
+    resetElement(element, commandStack);
```
Comment (MaxTru) [inline src/provider/zeebe/properties/BusinessRuleImplementationProps.js:79]: The undo/redo chain is broken if we have this as additional command.

If you for example:
1. Switch to DMN
2. Then switch to Job Worker
3. Then Undo
4. => will result in "none", where it should result in "DMN"

Can you please:
* Execute this within 1 command
* add test case for this bug

**Review by MaxTru — CHANGES_REQUESTED** [review]

File: src/provider/zeebe/properties/BusinessRuleImplementationProps.js [hunk 3]
```diff
@@ -135,3 +135,73 @@ function getCalledDecision(element) {
...
+    removeExtensionElements(element, businessObject, taskDefintion, commandStack);
+  }
+
+  if (calledDecision) {
+    removeExtensionElements(element, businessObject, calledDecision, commandStack);
+  }
+}
+
+
+export function updateExtensionElements(element, extensionElementToAdd, bpmnFactory, commandStack) {
```
[the remark this answers is not in the document]
Comment (smbea) [reply src/provider/zeebe/properties/BusinessRuleImplementationProps.js:154]: No, sorry. It was leftover from adapting an existing function in `ExtensionElementsUtil`. It's a specific case I haven't seen use for elsewhere so I'll just remove the `export`.

[the remark this answers is not in the document]
Comment (smbea) [reply src/provider/zeebe/properties/BusinessRuleImplementationProps.js]: done

**Review by MaxTru — APPROVED** [review]
Just removed an unnecessary empty line. 

:wave:

**Pull request merged**